lib/sufia/export/converter.rb

```diff
@@ -0,0 +1,23 @@
+module Sufia
+  module Export
+    # a base class to convert an ActiveFedora object that contains permissions and allow for pretty json.
+    #
+    class Converter
+      # overrides to_json to optionally allow for a pretty version of the json to be outputted
+      #
+      # @param [Boolean] pretty pass true to output formatted json using pretty_generate
+      def to_json(options = nil)
```

Comment:
Why set `options` to `nil` here? Then you have to check if it's `#blank?` on the next line. You could make this less ask-y and more tell-y by doing this:

```ruby
def to_json(options = {})
  json = super
  return json unless options[:pretty]
  JSON.pretty_generate(JSON.parse(json))
end
```

I don't care strongly about this, though. Are you inclined to care about that, or :shipit:?